openwisp_utils/utils.py

```diff
@@ -11,6 +11,14 @@ def get_random_key():
     return get_random_string(length=32)
 
 
+def add_menu_items(items, name_menu='OPENWISP_DEFAULT_ADMIN_MENU_ITEMS'):
```

Comment:
what about `register_menu_items`? This is also consistent with similar functions we added in the monitoring and notifications modules.